src/engine/source/builder/builders/stageBuilderCheck.cpp

```diff
@@ -89,10 +89,10 @@ base::Expression stageBuilderCheckExpression(const std::any& definition,
         // TODO: handle rest of operators
         else if (syntax::FUNCTION_HELPER_ANCHOR == term[0])
         {
-            auto pos1 = term.find("/");
+            auto pos1 = term.find('/');
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#performance

'find' called with a string literal consisting of a single character; use the more effective overload accepting a character